lib/Backends/NNPI/tests/NNPIOperatorTest.cpp

```diff
@@ -280,6 +277,7 @@ struct BlacklistInitializer {
              TestBlacklist::AnyDeviceAnyEngine},
             {"ChannelwiseQuantizedGroupConvolutionNonZero/0",
              TestBlacklist::AnyDeviceSWEngine},
+            {"BatchBoxCox_Float16/0", TestBlacklist::AnyDeviceAnyEngine},
```

Comment:
This is no longer the name of a test (we created Small/Medium/Large versions) -- it can be removed.